tests/test_defects.py

```diff
@@ -140,6 +140,17 @@ def check_maskBlocks(self, inputDefects, expectedDefects):
         for boxInput in expectedDefects:
             self.assertIn(boxInput, boxesMeasured)
 
+        # Union set of input and expected defects, as boxes.
```

Comment:
This is not a set, it is a list; did you mean for it to be a set? I don't remember if BBox is hashable.

I think you should move a version of the comment below up to here. Maybe something like this?

```
# Check that the code did not mask anything outside extra by
# looking in both the input list and "expanded-column" list.
```

Reply:
BBox is not hashable. We really have to consider how much of this code should really be inside `Defects`  -- the fact that internally it's all `BBox` is really an implementation detail.  This feels like it's not code that is specific to cp_pipe but more generally something that is comparing Defects.

Reply:
Let's not consider that on this ticket. :-)

If this is functional, let's clean up that comment and merge it. If you think we need to consider re-homing it, please file another ticket to cover that.

Reply:
Thanks, @parejkoj. I just had in mind the regular definition of a set in naïve set theory as a collection of objects, nothing technical. I'll use your suggested comment instead.